src/api/helpers/send-email-confirm-email-address/index.ts

```diff
@@ -29,7 +30,15 @@ const send = async (context: Context, urlOrigin: string, accountId: string): Pro
 
     let latestVerificationHash = '';
 
-    if (account.verificationHash) {
+    const verificationHasExpired = dateIsInThePast(account.verificationExpiry);
+
+    /**
+     * if the account already has a verificationHash and if it has not expired
+     * then set latestVerificationHash as account's verification hash
+     * else, generate new hash and update account
+     * if an account has expired verification, then new hash and verificationExpiry will be generated
```

Comment:
Can we match the documentation style as in the "else" on line 43? 1) ... 2) ... etc, 

I don't think we need to mention the "else" functionality before the "else"